sprocs/assessment_instances_select_log.sql

```diff
@@ -340,7 +340,13 @@ BEGIN
                 WHERE
                     pvl.assessment_instance_id = ai_id
                     AND pvl.page_type = 'studentInstanceQuestion'
-                    AND pvl.authn_user_id = ai.user_id
+                    AND (pvl.authn_user_id = ai.user_id
+                         OR EXISTS (SELECT 1
+                                    FROM group_logs gl
```

Comment:
Nit for consistency:
```suggestion
                                    FROM group_logs AS gl
```